rebar.config

```diff
@@ -9,5 +9,7 @@
     % {".*", "CFLAGS", "$CFLAGS -g -Wall -Werror -fPIC"}
 
     % Production compilation
-    {".*", "CFLAGS", "$CFLAGS -Wall -Werror -DNDEBUG -O3"}
+    {"(linux|solaris|darwin|freebsd)", "CFLAGS", "$CFLAGS -Wall -Werror -DNDEBUG -O3"},
+    {"win32", "CFLAGS", "$CFLAGS /O2 /DNDEBUG /Wall"}
```

Comment:
Windows doesn't have a -O3 flag for MSVC, only -O2.

You don't want warnings-as-errors, the code will never successfully compile. We do a lot of naughty things and I don't want cleanup of that code on all platforms to block our release.